Rename edge-messaging to simplex messaging

efim-poberezkin opened the pull request:
closes #10

epoberezkin commented on graph-chat.md:
```diff
@@ -19,16 +19,16 @@ TODO
 
-Majority of chat scenarios requires duplex (bi-directional) connections between participants. Graph-chat protocol uses multiple unidirectional connections (graph edges) created on multiple edge-messaging servers to implement duplex connections.
+Majority of chat scenarios requires duplex (bi-directional) connections between participants. Graph-chat protocol uses multiple unidirectional connections (simplexes) created on multiple simplex messaging servers to implement duplex connections.
 
```
Not sure we need to name them - can we just drop? We can just call them "simplex connections" or where it needs to be clarified in the beginning - "simplex (unidirectional) connections". Previously it was to justify the name really.

epoberezkin commented on graph-chat.md:
```diff
@@ -19,16 +19,16 @@ TODO
-  - `CBi` - Bob's edge-messaging connection number `i` (out of `n`) allowing Alice to send messages to Bob.
+- unidirectional simplex messaging connections:
+  - `CAi` - Alice's simplex messaging connection number `i` (out of `n`) allowing Bob to send messages to Alice.
+  - `CBi` - Bob's simplex messaging connection number `i` (out of `n`) allowing Alice to send messages to Bob.
```
we can just call them "simplex connection"

epoberezkin commented on graph-chat.md:
```diff
@@ -54,26 +54,26 @@ The following symbols are used below:
 
 1. Alice's app initiates duplex connection:
-   1. it creates `n` unidirectional connections ("edges") `CAi` (step 1 in [edge-messaging][3]) that are defined by:
+   1. it creates `n` unidirectional connections (simplexes) `CAi` (step 1 in [simplex messaging][3]) that are defined by:
```
same as above

epoberezkin commented on readme.md:
```diff
@@ -101,19 +101,19 @@ Client apps should provide the following features:
 
-1. [edge-messaging protocol][6] - a low level generic messaging protocol that defines establishing and using a unidirectional connection ("graph edge") between chat participants on a single server. While this protocol is designed to support graph-chat client protocol (below), it can be used for other messaging scenarios, not limited to chats.
-2. [edge-messaging protocol implementation][7] - requirements to clients and servers implementing edge-messaging protocol, including:
+1. [simplex messaging protocol][6] - a low level generic messaging protocol that defines establishing and using a unidirectional connection (simplex) between chat participants on a single server. While this protocol is designed to support graph-chat client protocol (below), it can be used for other messaging scenarios, not limited to chats.
```
same as above - suggest dropping "simplex" as noun

epoberezkin commented on readme.md:
```diff
@@ -101,19 +101,19 @@ Client apps should provide the following features:
-   - other requirements for edge-messaging servers.
-3. [graph-chat protocol][8] - a high level generic chat protocol for client applications (graph vertices) that communicate via connections (graph edges) created using edge-messaging protocol. This protocol defines connection and message types and semantics for:
+   - other requirements for simplex messaging servers.
+3. [graph-chat protocol][8] - a high level generic chat protocol for client applications (graph vertices) that communicate via connections (simplexes) created using simplex messaging protocol. This protocol defines connection and message types and semantics for:
```
same

efim-poberezkin commented:
> the main comment is about using "simplex" as noun - I think we can just call it "simplex connection", "simplex" as noun sounds weird here as it already has a different meaning.

agreed, I've changed wordings accordingly, also I've changed "unidirectional [simplex]" to "simplex" across text except for a few places where the concept is being first introduced
